sky/utils/schemas.py

```diff
@@ -762,7 +771,7 @@ def get_config_schema():
                     'high_availability': {
                         'type': 'boolean',
                     },
-                    'autostop': autostop_schema,
+                    'autostop': _AUTOSTOP_SCHEMA,
```

Comment:
May need to print a deprecation warning when this is specified.

Reply:
Added this.